### PR TITLE
Updated with latest API features

### PR DESCRIPTION
**Pull request by daveirwin1**
Added process groups, Azure metadata and dynamic machine groups.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by anuchandy** [comment]
Addition of properties to an existing api version is breaking changes. Adding ARM to take a look.

**Comment by anuchandy** [comment]
@ravbhatnagar could you please take a look?

File: specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json [hunk 1]
```diff
@@ -1867,18 +1930,39 @@
           "type": "object",
           "description": "Resource properties.",
           "properties": {
+            "groupType": {
+              "type": "string",
+              "readonly": true,
```
Comment (anuchandy) [inline specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json]: `readOnly` (use camelcase "O")

File: specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json [hunk 2]
```diff
@@ -1733,6 +1787,11 @@
               },
               "description": "The inferred role of this process based on its name, command line, etc."
             },
+            "group": {
+              "type": "string",
+              "readonly": true,
```
Comment (anuchandy) [inline specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json]: `readOnly` (use camelcase "O")

File: specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json [hunk 3]
```diff
@@ -1720,6 +1773,7 @@
             },
             "role": {
               "type": "string",
+              "readonly": true,
```
Comment (anuchandy) [inline specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json]: `readOnly` (use camelcase "O")

File: specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json [hunk 4]
```diff
@@ -1673,6 +1721,10 @@
             "hypervisor": {
               "$ref": "#/definitions/HypervisorConfiguration",
               "description": "Hypervisor-related configuration. Present only when 'virtualizationState' is `hypervisor`."
+            },
+            "hosting": {
+              "$ref": "#/definitions/HostingConfiguration",
+              "description": "Hosting-related configuration. Present if hosting information is discovered for the VM."
```
Comment (anuchandy) [inline specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json]: From the comment "Present if hosting information is discovered for the VM" it seems this property is readonly, if so add `readOnly` attribute

**Review by anuchandy — CHANGES_REQUESTED** [review]

**Comment by anuchandy** [comment]
@daveirwin1 please take a look at the comments. 

Are you from the operation insight service team? Was trying to find your contact so that we can include you in the email threads with ARM team, let me know how can we locate you in outlook.

**Comment by daveirwin1** [comment]
@anuchandy My alias is dairwin.  I'm a PM on the Service Map team.  I'm not sure what the Operation Insight service team is.

**Comment by anuchandy** [comment]
@daveirwin1 thanks. Looped you into the email thread with ARM.

**Comment by anuchandy** [comment]
Answering my [question](https://github.com/Azure/azure-rest-api-specs/pull/2209#issuecomment-356383815) based on the offline follow-up with @daveirwin1 

This is the API for `Service Map`, which is a solution in Operations Management Suite (OMS), formerly known as `Operational Insights`.  OMS is now mostly known as `Log Analytics`. `Service Map` is a part of `OMS/Operational Insights/Log Analytics` but it’s not the whole thing.

**Comment by anuchandy** [comment]
@daveirwin1, did you get a chance to look into above comments?

**Comment by daveirwin1** [comment]
Yes, I talked to our devs and they're getting me an updated json file.
I'll update the PR when I get it.

On Thu, Jan 11, 2018 at 2:15 PM, Anu Thomas Chandy <notifications@github.com

**Comment by ravbhatnagar** [comment]
@daveirwin1  @anuchandy  - is this ready for ARM to take a look?

**Comment by anuchandy** [comment]
@ravbhatnagar not yet, @daveirwin1 is waiting to get updated spec from the dev team

**Comment by anuchandy** [comment]
@daveirwin1 did you get a chance to update the spec? I'm removing `waitForARMFeedback` since spec needs to be updated in-order to kick of ARM review.

**Comment by daveirwin1** [comment]
No, not yet.

On Fri, Jan 19, 2018 at 2:09 PM, Anu Thomas Chandy <notifications@github.com

**Comment by anuchandy** [comment]
@daveirwin1 closing this due to no activity for last 5 months, feel free to open a new PR when you are ready.

**Pull request closed without merging**